src/other/Table.js

```diff
@@ -178,6 +186,20 @@
         ;
         th.order();
 
+        //Disable Pagination if a widget is in the data
+        if (context.pagination()) {
+            for (var iDx1 = 0; iDx1 < this._db._data.length; iDx1 ++) {
+                var d = this._db._data[iDx1];
+                for (var iDx2 = 0; iDx2 < d.length; iDx2 ++) {
+                    if (d[iDx2] instanceof Widget) {
+                        context.pagination(false);
+                        break;
+                    }
+                }
+                if (context.pagination() === false) {break;}
+            }
+        }
```

Comment:
And just let the pagination break?  That's fine with me, it will hopefully be updated soon and it will be a non-issue.